Tests/configure_and_test_integration_instances.py

```diff
@@ -186,6 +189,15 @@ class Build(ABC):
     DEFAULT_SERVER_VERSION = '99.99.98'
 
     #  END CHANGE ON LOCAL RUN  #
+    @staticmethod
+    def get_secret_service_account_config() -> str:
+        print('##################################')
+        print('##################################')
+        print('##################################')
+        print(os.environ['GSM_SERVICE_ACCOUNT'])
+        with open(os.environ['GSM_SERVICE_ACCOUNT']) as f:
```

Comment:
remove/replace infra code in content repo related to secrets.
download_demisto_conf.sh